playwright-e2e/package.json

```diff
@@ -15,7 +15,7 @@
   },
   "devDependencies": {
     "@faker-js/faker": "^8.3.1",
-    "@playwright/test": "^1.41.0",
+    "@playwright/test": "^1.50.0",
```

Comment:
this update is so that I could use the toMatchAriaSnapshot() method which was introduced in Playwright version 1.49 from what I understand